src/main/java/com/sprint/mission/discodeit/entity/ReadStatus.java

```diff
@@ -0,0 +1,18 @@
+package com.sprint.mission.discodeit.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Getter
+@Setter
+public class ReadStatus {
```

Comment:
만약 JPA를 도입했다면 여기에도 `@Entity` 애노테이션을 붙여야할 것 같네요.  

```suggestion
@Getter
@Setter
@Entity
public class ReadStatus {
```